### ISSUE TITLE
Reference layers as in Aseprite

### ISSUE THREAD

**Issue by freetz13** [LibreSprite/LibreSprite-proposals#42]
Ability to use some image as a reference layer, which I can shrink to canvas size without loosing its resolution.

**Comment by Zughy** [comment]
Actually, I don't know how much it can be necessary, when this can already been done.
![imagen](https://user-images.githubusercontent.com/63455151/143659685-9743bf4b-234f-4ec3-baff-195a635765be.png)

Can you think of any particular usecase aside tracing (which is not a really nice thing to do anyway)?

**Comment by freetz13** [comment]
> Can you think of any particular usecase aside tracing

Well, I use proprietary Aseprite exactly for tracing. Why do you think it's not a nice thing?

I'm learning to draw pixel art, and I made this picture, thanks to Aseprite's reference layers feature:

![lain](https://user-images.githubusercontent.com/7419347/143669136-0a098be8-ebc5-430a-815d-e88678d20f92.png)

I just can't draw that by hand. I would be happy if I see reference layers in LibreSprite.

**Comment by Zughy** [comment]
Well, tracing doesn't help learning how to draw

**Comment by freetz13** [comment]
> tracing doesn't help learning how to draw

No, it helps. At least for complete beginners as I am. It trains drawing skill.

**Comment by therealrobster** [comment]
For feedback.  The BEST I've ever seen at diong a reference image is Pixly which can be found http://meltinglogic.com/projects/pixly and https://play.google.com/store/apps/details?id=com.meltinglogic.pixly  Seriously I recommend everyone try their reference system.  It's beautiful and works so well.  I hope that helps

**Comment by princessmadllama** [comment]
> 

I could use the feature for 3D work.
Example if I export a UV map from 3D software I could then use it as a a layer reference for when drawing my pixel art textures.

**Comment by MysticStar32** [comment]
> Actually, I don't know how much it can be necessary, when this can already been done. ![imagen](https://user-images.githubusercontent.com/63455151/143659685-9743bf4b-234f-4ec3-baff-195a635765be.png)
> 
> Can you think of any particular usecase aside tracing (which is not a really nice thing to do anyway)?

How do you do that, cant quite figure it out

**Comment by cesarblender** [comment]
any updates on this?? this feature is very useful for newbies, and its helpful to have a reference

**Comment by bijsven** [comment]
Any updates?

**Comment by lvasix** [comment]
tracing my own art is what I'm trying to do, I have a rough sketch that I would love to be able to trace with libresprite, tracing does not equal stealing art. This would be such a useful feature

**Comment by ItsFoxy87** [comment]
I'm a little late, but for anyone else having an issue with this, you could use OnTopReplica from Github which will let you display any image over any application (in this case Libresprite) for tracing, it even lets you make the window transparent too. This can be a good alternative until Libresprite devs properly implement this, if they ever update the program.

**Comment by freetz13** [comment]
> OnTopReplica

Thank you, but it works on Windows only. What about linux guys? I know, composite managers and all. But I would expect this feature from LibreSprite itself, not from DE.
